ament_cmake_core/cmake/index/ament_index_get_resources.cmake

```diff
@@ -52,10 +52,17 @@ function(ament_index_get_resources var resource_type)
   list(INSERT paths_to_search 0 "${CMAKE_INSTALL_PREFIX}")
   set(all_resources "")
   foreach(path IN LISTS paths_to_search)
+    set(resource_index_path "${path}/share/ament_index/resource_index")
     file(GLOB resources
-      RELATIVE "${path}/share/ament_index/resource_index/${resource_type}"
-      "${path}/share/ament_index/resource_index/${resource_type}/*")
-    list_append_unique(all_resources ${resources})
+      RELATIVE "${resource_index_path}/${resource_type}"
+      "${resource_index_path}/${resource_type}/*")
```

Comment:
Newer versions of `file(GLOB` have the option to pass `LIST_DIRECTORIES false` directly. Please add a todo above this to use this when we upgrade the code soon to rely on CMake 3.5.